docs/reference_dataset/pipelines/hello-world-root-depls-init-generated.yml

```diff
@@ -13,7 +13,7 @@ resources:
   type: slack-notification
   source:
     url: ((slack-webhook))
-- name: fe-int-micro-for-micro-depls
+- name: concourse-target
```

Comment:
This value is now consistent with https://github.com/orange-cloudfoundry/cf-ops-automation/blob/04943eb5e580281cb3e1a28730b312e1521a86eb/concourse/tasks/fly_execute_commands.yml#L37